README.md

```diff
@@ -35,13 +31,20 @@ dependencies {
 
 ### How to publish to JCenter
 
-1. Upload from your local.
+#### GitHub
 
-    ```sh
-    $ export BINTRAY_USER=wantedly-dev
-    $ export BINTRAY_KEY=1234567890
-    $ ./gradlew clean bintrayUpload
-    ```
+You can publish this library using `publish` workflow from GitHub Actions.
 
-1. Go to https://bintray.com/wantedly-dev/maven/com.wantedly%3Awantedly-maven-repository
-1. Click Publish button
+You must change the artifact version before running `publish` workflow to publish the new version.
+Otherwise, `publish` workflow will fail.
+
+#### Local
+
+```sh
+$ export ORG_GRADLE_PROJECT_OSSRH_PASSWORD=XXXXXX
+$ export ORG_GRADLE_PROJECT_OSSRH_USERNAME=XXXXXX
+$ export ORG_GRADLE_PROJECT_SIGNING_KEY=XXXXXX
+$ export ORG_GRADLE_PROJECT_SIGNING_KEY_ID=XXXXXX
+$ export ORG_GRADLE_PROJECT_SIGNING_PASSWORD=XXXXXX
+$ ./gradlew clean publish
```

Comment:
💡 Can you add an explanation on how to get these values?

Reply:
Added about the secrets. 9f4b0f1